ms/MSOper/MSMetaData.cc

```diff
@@ -3399,6 +3399,8 @@ void MSMetaData::_computeScanAndSubScanProperties(
     uInt chunkSize = nrows/nchunks;
     if (nrows % nchunks > 0) {
         ++chunkSize;
+        // integer division
+        nchunks = nrows/chunkSize + 1;
     }
```

Comment:
but thats just the same special case written differently, so yeah not so useful

is the `++chunkSize` still neeeded? I read that as the nchunks++ original so I didn't notice the nchunk issue